packages/hub/routes/email-card-drop-requests.ts

```diff
@@ -60,6 +68,65 @@ export default class EmailCardDropRequestsRoute {
     ctx.type = 'application/vnd.api+json';
     return;
   }
+
+  async post(ctx: Koa.Context) {
+    if (!ensureLoggedIn(ctx)) {
+      return;
+    }
+
+    let email = ctx.request.body.data.attributes.email;
+
+    let hash = crypto.createHash('sha256');
+    hash.update(email);
+    let emailHash = hash.digest('hex');
+
+    let verificationCode = (crypto.randomInt(1000000) + '').padStart(6, '0');
+
+    if (isEmail(email)) {
+      const emailCardDropRequest: EmailCardDropRequest = {
+        id: shortUuid.uuid(),
+        ownerAddress: ctx.state.userAddress,
```

Comment:
I don't think we should use `ctx.state.userAddress`. CS-3669 describes the payload that will be sent to this endpoint:

> Hub auth token, email address, and EOA address

Reply:
actually, thinking about this more - it feels better to use the address from the auth token to avoid a situation where we end up with a card drop request where the address from the request payload doesn't match the address in the auth token.